src/Keboola/DbExtractor/Test/ExtractorTest.php

```diff
@@ -12,7 +12,6 @@
 
 class ExtractorTest extends TestCase
 {
-    public const CONFIG_FORMAT_YAML = 'yaml';
     public const CONFIG_FORMAT_JSON = 'json';
```

Comment:
to jeste pouzivame?  nemuzem ho taky vyhodit?

Reply:
máš pravdu - smazal jsem ho